aws-ecs/10-vpc.tf

```diff
@@ -0,0 +1,29 @@
+data "aws_availability_zones" "available" {}
+
+module "vpc" {
+  count = var.create_vpc ? 1 : 0
+
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "3.19.0"
+
+  name             = "pvault-vpc"
+  cidr             = "10.0.0.0/16"
```

Comment:
I think we should export this as variable as well. CIDR allocation organization may matter to certain users.